app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -75,8 +75,8 @@ class Scene extends React.Component {
       element: this.scene.current,
       engine: this.engine,
       options: {
-        width: window.innerWidth,
-        height: window.innerHeight,
+        width: this.state.width,
+        height: this.state.height,
```

Comment:
I think these `width` and `height` changes can be their own commit.

Reply:
Why are width and height in state?